example.c

```diff
@@ -0,0 +1,79 @@
+// Includes are not automatically sorted
+#include <a.h> // Comments
+#include "c.h" // Are
+#include <b.h> // ALigned
+
+// declarations:
+
+int SimpleFunction(int a, int b, int c);
+
+void MoreParameters(
+    long long one_number, const char *one_string, const void *some_data);
+
+void ManyParameters(
+    long long one_number,
+    const char *one_string,
+    const void *some_data,
+    const void *more_data);
+
+// definitions:
+
+int SimpleFunction(int a, int b, int c) { return a + b + c; }
+
+int MoreParameters(
+    long long one_number, const char *one_string, const void *some_data)
+{
+    return (int)one_number + (int)one_string + (int)some_data;
+}
+
+int ManyParameters(
+    const long long one_number,
+    const char *const one_string,
+    ARG_UNUSED const void *const some_data,
+    ARG_UNUSED const void *const more_data);
+{
+    return (int)one_number + (int)one_string;
+}
+
+int main(int argc, char **argv)
+{
+    // Most of the time function calls and assignments should fit on one line:
+    int a_number = SimpleFunction(1, 2, 3);
+
+    // Assignments are not aligned:
+    int a = 0;
+    int abc = 10;
+    {
+        // Sometimes the function call and parameter can fit on the next line:
+        int complicated_result =
+            MoreParameters(a_number, NULL, (const void *)NULL);
+
+        const char *const string_with_data = "data";
+
+        // For long lists of arguments, put each on separate line
+        // Don't align with open brace
+        ManyParameters(
+            complicated_result * complicated_result,
+            string_with_data,
+            (const void *)string_with_data,
+            (const void *)string_with_data);
+
+        // If spacing and braces according to style guidelines:
+        if (a_number == 10)
+        {
+            return 1;
+        }
+
+        // Switch cases must be on separate lines:
+        switch (a_number)
+        {
+        case 9:
+            return 1;
```

Comment:
The case lines not being indented looks strange to me.

Reply:
This is the way it is through most of our code-base. Picked up from [Linux Kernel coding style](https://www.kernel.org/doc/html/latest/process/coding-style.html). It also avoids the "double indent" of case contents, so I think I'll stick with this one.